src/python/turicreate/toolkits/one_shot_object_detector/util/_error_handling.py

```diff
@@ -6,8 +6,13 @@
 
 import turicreate as _tc
 import turicreate.toolkits._internal_utils as _tkutl
+from turicreate.toolkits._main import ToolkitError as _ToolkitError
 
-def check_one_shot_input(data, target):
+def check_one_shot_input(data, target, backgrounds):
+    if backgrounds is not None and not(isinstance(backgrounds, _tc.SArray)):
+        raise TypeError("'backgrounds' must be of type SArray.")
```

Comment:
It could also be `None`. I would say `"'backgrounds' must be None or an SArray."`

Reply:
I will change it. Sounds better.